app/models/cohort_columns/last_homeless_visit.rb

```diff
@@ -1,4 +1,4 @@
-###
```

Comment:
while minor, we'll need the extra #.  There's a script we use to update these each year, and if they don't match exactly, it adds duplicates.

Reply:
✅ Glad to know about the script. I figured there must be something like that, and that change was inadvertent.